FactoryWeather/App/VIP Settings/DataSource/SettingsDataSource.swift

```diff
@@ -35,6 +35,10 @@ extension SettingsDataSource {
         self.settings = settings
         buildSections()
     }
+    
+    func sectionIndex(forCase caseType: SettingsSection) -> Int? {
```

Comment:
We could improve the naming here a bit (although, this is perfectly okay): 
`index(for section: SettingsSection`.

Your call 🙂